nodejs/week4/spam-detector/index.js

```diff
@@ -0,0 +1,89 @@
+class Email {
+	constructor(subject, body) {
+		this.subject = subject;
+		this.body = body;
+	}
+}
+
+class SpamDetector {
+	badWordCheck(email) {
```

Comment:
Try indicate what a function will do. Fx return a boolean. badWordCheck is difficult to figure out what does

```suggestion
	hasBadWord(email) {
```